src/com/itacademy/eshop/product/Toy.java

```diff
@@ -0,0 +1,9 @@
+package com.itacademy.eshop.product;
+
+import com.itacademy.eshop.product.types.Category;
+
+public abstract class Toy extends Product {
```

Comment:
Same as other places. You misunderstood why we are using abstract class. Without additional fields or unimplemented methods it serves no purpose. Check example with Computer and Electronics and you will see a difference